templates/validator/charts.html

```diff
@@ -190,12 +190,6 @@
                                     enabled: false
                                 },
                                 pointInterval: 24 * 3600 * 1000,
-                                dataGrouping: {
```

Comment:
**issue:** I don't think this leads to an x-axis labeling as desired.
Now the marks on the x-axis are placed when a new calendar day starts and don't match with the bars.
The epochs are also still the same with the first entry on the x-axis on prater is "24. Mar Epoch 84".

However maybe this design is acceptable, if you want it can be discussed.